README.md

```diff
@@ -94,19 +106,19 @@ docker run --network=host --gpus=all roboflow/roboflow-inference-server-gpu:late
 ```bash
 docker run -p 9001:9001 roboflow/roboflow-inference-server-arm-cpu:latest
 ```
-  
+
 - Run on Nvidia GPU with TensorRT Runtime:
 
 ```bash
 docker run --network=host --gpus=all roboflow/roboflow-inference-server-trt:latest
 ```
-  
+
 - Run on Nvidia Jetson with JetPack `4.x`:
 
 ```bash
 docker run --privileged --net=host --runtime=nvidia roboflow/roboflow-inference-server-trt-jetson:latest
 ```
-  
+
 - Run on Nvidia Jetson with JetPack `5.x`:
```

Comment:
`NVIDIA` should be all in capital letters.

Reply:
Done